### ISSUE TITLE
bug: Response

### ISSUE THREAD

**Issue by PandaWorker** [oven-sh/bun#8526]
### What version of Bun is running?

1.0.25+a8ff7be64

### What platform is your computer?

Darwin 23.1.0 arm64 arm

### What steps can reproduce the bug?

```ts
import * as undici from 'undici';

const { statusCode: status, body, headers } = await undici.request('https://api.ipify.org');
const response = new undici.Response(body, { status, body, headers });

console.dir(await response.text());
```

```
> bun run 1.mjs
[object EventEmitter]

> node 1.mjs
87.21.3.23
```

**Comment by pzmarzly** [comment]
Seems to have been fixed, works for me on Bun 1.1.20 (Darwin 23.6.0 arm64).

**Comment by RiskyMH** [comment]
I also can't seem to reproduce, if this is still an issue @PandaWorker let me know and I can re-open.
